bigchaindb-on-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,252 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "admin_username": {
+            "defaultValue": null,
+            "type": "string",
+            "metadata": {
+                "description": "The admin username"
+            }
+        },
+        "admin_password": {
+            "defaultValue": null,
+            "type": "securestring",
+            "metadata": {
+                "description": "The admin password"
+            }
+        },
+        "dns_label_prefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+            }
+        },
+        "virtual_machine_size": {
+            "defaultValue": "Standard_D2",
+            "type": "string",
+            "metadata": {
+                "description": "The virtual machine size"
+            }
+        },
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+            },
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/bigchaindb-on-ubuntu/"
+        },
+        "_artifactsLocationSasToken": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+            },
+            "defaultValue": ""
+        }
+    },
+    "variables": {
+        "location": "[resourceGroup().location]",
+        "storageAccount": "[concat(uniquestring(resourceGroup().id), 'storage')]",
+        "vmName": "bigchaindb",
+        "nicName": "BDBNic",
+        "publicIpName": "BDBPublicIp",
+        "vnetName": "BDBVNet",
+        "nsgName": "BDBNSG",
+        "OSDiskName": "bigchaindb_os_disk",
+        "fileUris": "[concat(parameters('_artifactsLocation'), '/scripts/init.sh', parameters('_artifactsLocationSasToken'))]",
+        "vnetID": "[resourceId('Microsoft.Network/virtualNetworks', variables('vnetName'))]",
+        "subnetRef": "[concat(variables('vnetID'), '/subnets/', 'default')]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Compute/virtualMachines",
+            "name": "[variables('vmName')]",
+            "apiVersion": "2015-06-15",

```

Comment:
can you use 2016-03-03 here since the other resources are using a later version?